.github/workflows/actions.yml

```diff
@@ -81,7 +81,10 @@ jobs:
         KERAS_CV_MULTI_BACKEND: true
         KERAS_BACKEND: jax
       run: |
-        pytest keras_cv/models/backbones/resnet_v1 --durations 0
+        pytest keras_cv/models/backbones/resnet_v1 \
+               keras_cv/models/backbones/resnet_v2 \
+               keras_cv/models/classification/image_classifier_test.py \
+               --durations 0
```

Comment:
Since we have a relatively small set of tests for now for Keras Core, maybe let's add --run_large just to maximize coverage until we're ready to run our GCB tests on Keras Core

Reply:
Added `run_large` and also `run_extra_large`. For `EfficientNet`, all the preset tests are marked as `extra_large` so I think it would be good to cover them as long as they don't take too long to finish.